### DISCUSSION TITLE
Name of the Game

### DISCUSSION THREAD

**Discussion by bf-bonkus** [bf-bonkus/breadiquette#21] [Ideas]
### Overview
Our game is currently called "fbla-game-2024" everywhere and this needs to be changed ASAP.
### Suggestions
The best I can come up with rn is "Breadiquette," but this isn't final. Reply if this name sounds good or if you have a better idea.

**Comment by RuinKnight** [comment]
I like this name

**Comment by Mandbarnot** [comment]
Why a breadlabel?
